recipes/b2b-utils/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "b2b-utils" %}
+{% set version = "0.017" %}
+{% set sha256 = "99fe213d6404076b5451f441413b628df3357ddb1c9e9c5458974addac02d6f0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/jvolkening/b2b-utils/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - make
+    - perl
+    - perl-bioperl
+    - perl-biox-seq >=0.008006
+    - perl-dbi
+    - perl-file-compare
+    - perl-list-moreutils
+    - perl-module-build
+    - perl-perlio-gzip
+    - perl-test-more
+    - perl-test-pod
+    - bwa
```

Comment:
why is this a host requirement but not a runtime one?

Reply:
Already noted in `meta.yaml` extra::notes: extra binaries used by some tools are included as host requirements so that the testing suite will be run for those tools. However, since not all tools need them, final installation is left up to the end user. In particular, requiring medaka for installation increases the footprint quite a bit.

Reply:
Oha, thanks for putting this into the comments, even if I have not seen it.